src/modelframe.jl

```diff
@@ -53,11 +53,12 @@ function _nonmissing!(res, col)
     end
 end
 
-function _nonmissing!(res, col::CategoricalArray{>: Missing})
-    @inbounds for (i, el) in enumerate(col.refs)
-        res[i] &= el > 0
-    end
-end
+# TODO: find another way to optimize this without taking a dependency
```

Comment:
Just remove this thanks to https://github.com/JuliaData/CategoricalArrays.jl/pull/218.

Reply:
does this mean that we need to change these methods to use broadcast instead of doing a loop?